### PR TITLE
bw: fix fetch balance

### PR DESCRIPTION
**Pull request by brandsimon**
currencyId is a int, but since currencies_by_id is an object, python needs this accessor to be a string.

### CODE REVIEW

File: python/ccxt/base/exchange.py [hunk 1]
```diff
@@ -1622,6 +1622,7 @@ def parse_orders(self, orders, market=None, since=None, limit=None, params={}):
 
     def safe_currency_code(self, currency_id, currency=None):
         code = None
+        currency_id = str(currency_id)
```
Comment (kroitor) [inline python/ccxt/base/exchange.py]:
```Python
mbp:ccxt.pro igorkroitor$ python3
Python 3.7.2 (v3.7.2:9a3ffc0492, Dec 24 2018, 02:44:43)
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> str(None)
'None'
```

Comment (kroitor) [reply python/ccxt/base/exchange.py]: I think we should swap the outer `if`s, what do you say?

Comment (brandsimon) [reply python/ccxt/base/exchange.py]: Great thinking, thank you!

Comment (kroitor) [reply python/ccxt/base/exchange.py]: Thx to you for the fix, merging shortly! )

**Pull request merged**